pkg/cloud/openstack/openstack.go

```diff
@@ -104,6 +108,15 @@ func updateLoadBalancerSection(loadBalancer *ini.Section, network *configv1.Netw
 			enabledKey.SetValue("false")
 		}
 	}
+	// Enable manage-security-groups by default if not overriden already
+	_, err = loadBalancer.GetKey("manage-security-groups")
```

Comment:
Is the intention that, if it's already set, we don't care what it is, so won't overwrite? We don't enforce this to be true?

Reply:
This is indeed my intention to allow overriding it manually. I can think of some use cases when it makes sense:

* User doesn't care, then having the full range of NodePorts opened on master/worker SG puts less stress on Neutron.
* User has a Neutron plugin that is bad at SGs (happened in the past) and prefers to limit the number of them.
* Opening an SG takes some time (for CPO to create it, for Neutron to actually enforce it) and that's unacceptable for user's workload.

I intend to create an extensive release note to go with this PR along with instructions how to override the new behavior if need be.